### PR TITLE
chore(rds): add missing AuroraPostgresEngineVersions 9.6.18, 9.6.19, 10.13 and 10.14

### PR DESCRIPTION
**Pull request by blimmer**
This PR adds missing engine versions. See inline comments for API descriptions.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-rds/lib/cluster-engine.ts [hunk 1]
```diff
@@ -422,6 +422,10 @@ export class AuroraPostgresEngineVersion {
   public static readonly VER_9_6_16 = AuroraPostgresEngineVersion.of('9.6.16', '9.6');
   /** Version "9.6.17". */
   public static readonly VER_9_6_17 = AuroraPostgresEngineVersion.of('9.6.17', '9.6');
+  /** Version "9.6.18". */
+  public static readonly VER_9_6_18 = AuroraPostgresEngineVersion.of('9.6.18', '9.6');
```
Comment (blimmer) [inline packages/@aws-cdk/aws-rds/lib/cluster-engine.ts:426]:
```json
{
    "DBEngineVersions": [
        {
            "Engine": "aurora-postgresql",
            "EngineVersion": "9.6.18",
            "DBParameterGroupFamily": "aurora-postgresql9.6",
            "DBEngineDescription": "Aurora (PostgreSQL)",
            "DBEngineVersionDescription": "Aurora PostgreSQL (Compatible with PostgreSQL 9.6.18)",
            "ValidUpgradeTarget": [
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "9.6.19",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 9.6.19)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": false
                },
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "10.13",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 10.13)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": true
                },
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "10.14",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 10.14)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": true
                }
            ],
            "ExportableLogTypes": [
                "postgresql"
            ],
            "SupportsLogExportsToCloudwatchLogs": true,
            "SupportsReadReplica": false,
            "SupportedEngineModes": [
                "provisioned"
            ],
            "SupportedFeatureNames": [],
            "Status": "available",
            "SupportsParallelQuery": false,
            "SupportsGlobalDatabases": false
        }
    ]
}
```

File: packages/@aws-cdk/aws-rds/lib/cluster-engine.ts [hunk 2]
```diff
@@ -422,6 +422,10 @@ export class AuroraPostgresEngineVersion {
...
+  /** Version "9.6.19". */
+  public static readonly VER_9_6_19 = AuroraPostgresEngineVersion.of('9.6.19', '9.6');
```
Comment (blimmer) [inline packages/@aws-cdk/aws-rds/lib/cluster-engine.ts:428]:
```json
{
    "DBEngineVersions": [
        {
            "Engine": "aurora-postgresql",
            "EngineVersion": "9.6.19",
            "DBParameterGroupFamily": "aurora-postgresql9.6",
            "DBEngineDescription": "Aurora (PostgreSQL)",
            "DBEngineVersionDescription": "Aurora PostgreSQL (Compatible with PostgreSQL 9.6.19)",
            "ValidUpgradeTarget": [
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "10.14",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 10.14)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": true
                }
            ],
            "ExportableLogTypes": [
                "postgresql"
            ],
            "SupportsLogExportsToCloudwatchLogs": true,
            "SupportsReadReplica": false,
            "SupportedEngineModes": [
                "provisioned"
            ],
            "SupportedFeatureNames": [],
            "Status": "available",
            "SupportsParallelQuery": false,
            "SupportsGlobalDatabases": false
        }
    ]
}
```

File: packages/@aws-cdk/aws-rds/lib/cluster-engine.ts [hunk 3]
```diff
@@ -434,6 +438,10 @@ export class AuroraPostgresEngineVersion {
   public static readonly VER_10_11 = AuroraPostgresEngineVersion.of('10.11', '10', { s3Import: true, s3Export: true });
   /** Version "10.12". */
   public static readonly VER_10_12 = AuroraPostgresEngineVersion.of('10.12', '10', { s3Import: true, s3Export: true });
+  /** Version "10.13". */
+  public static readonly VER_10_13 = AuroraPostgresEngineVersion.of('10.13', '10', { s3Import: true, s3Export: true });
```
Comment (blimmer) [inline packages/@aws-cdk/aws-rds/lib/cluster-engine.ts:442]:
```json
{
    "DBEngineVersions": [
        {
            "Engine": "aurora-postgresql",
            "EngineVersion": "10.13",
            "DBParameterGroupFamily": "aurora-postgresql10",
            "DBEngineDescription": "Aurora (PostgreSQL)",
            "DBEngineVersionDescription": "Aurora PostgreSQL (Compatible with PostgreSQL 10.13)",
            "ValidUpgradeTarget": [
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "10.14",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 10.14)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": false
                },
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "11.8",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 11.8)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": true
                },
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "11.9",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 11.9)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": true
                }
            ],
            "ExportableLogTypes": [
                "postgresql"
            ],
            "SupportsLogExportsToCloudwatchLogs": true,
            "SupportsReadReplica": false,
            "SupportedEngineModes": [
                "provisioned"
            ],
            "SupportedFeatureNames": [
                "s3Import",
                "Comprehend",
                "s3Export",
                "SageMaker"
            ],
            "Status": "available",
            "SupportsParallelQuery": false,
            "SupportsGlobalDatabases": true
        }
    ]
}
```

File: packages/@aws-cdk/aws-rds/lib/cluster-engine.ts [hunk 4]
```diff
@@ -434,6 +438,10 @@ export class AuroraPostgresEngineVersion {
...
+  /** Version "10.14". */
+  public static readonly VER_10_14 = AuroraPostgresEngineVersion.of('10.14', '10', { s3Import: true, s3Export: true });
```
Comment (blimmer) [inline packages/@aws-cdk/aws-rds/lib/cluster-engine.ts:444]:
```json
{
    "DBEngineVersions": [
        {
            "Engine": "aurora-postgresql",
            "EngineVersion": "10.14",
            "DBParameterGroupFamily": "aurora-postgresql10",
            "DBEngineDescription": "Aurora (PostgreSQL)",
            "DBEngineVersionDescription": "Aurora PostgreSQL (Compatible with PostgreSQL 10.14)",
            "ValidUpgradeTarget": [
                {
                    "Engine": "aurora-postgresql",
                    "EngineVersion": "11.9",
                    "Description": "Aurora PostgreSQL (Compatible with PostgreSQL 11.9)",
                    "AutoUpgrade": false,
                    "IsMajorVersionUpgrade": true
                }
            ],
            "ExportableLogTypes": [
                "postgresql"
            ],
            "SupportsLogExportsToCloudwatchLogs": true,
            "SupportsReadReplica": false,
            "SupportedEngineModes": [
                "provisioned"
            ],
            "SupportedFeatureNames": [
                "s3Import",
                "Comprehend",
                "s3Export",
                "SageMaker"
            ],
            "Status": "available",
            "SupportsParallelQuery": false,
            "SupportsGlobalDatabases": true
        }
    ]
}
```

**Review by skinny85 — APPROVED** [review]
Thanks for the contribution @blimmer !

**Pull request merged**